apps/jobs/models.py

```diff
@@ -81,11 +81,11 @@ class Submission(TimeStampedModel):
         upload_to=RandomFileName("submission_files/submission_{id}"), null=True, blank=True)
     submission_metadata_file = models.FileField(
         upload_to=RandomFileName("submission_files/submission_{id}"), null=True, blank=True)
-    execution_time_limit = models.PositiveIntegerField(default=300, db_index=True)
-    method_name = models.CharField(max_length=1000, null=True, db_index=True)
```

Comment:
@deshraj @taranjeet I think we can create a hash of method name as in future it will be easy for  challenge host to filter the results on the basis of `method name`.

Reply:
@RishabhJain2018 : As of now I think that this is not required. We will see if any need pops up in the future.

Reply:
@taranjeet I was talking of the same thing as @deshraj was talking in the below comment :sweat_smile: